cmd/kubectl/app/kubectl.go

```diff
@@ -33,6 +34,12 @@ Any salient changes here will need to be manually reflected in that file.
 func Run() error {
 	logs.InitLogs()
 	defer logs.FlushLogs()
+	i18n.LoadTranslations("kubectl", nil, []string{
```

Comment:
Can be a follow-up, but since you only care about the set of messages for one single language at each `kubectl` run, the language detection could happen here and you'd only load the language you care about.